images/linux/scripts/SoftwareReport/SoftwareReport.Android.psm1

```diff
@@ -0,0 +1,147 @@
+    return @(
+        @{
+            "Package" = "Android SDK Platform-Tools"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android SDK Platform-Tools"
+        },
+        @{
+            "Package" = "Android SDK Tools"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android SDK Tools"
+        },
+        @{
+            "Package" = "Android SDK Platforms"
+            "Version" = Get-AndroidPlatformVersions -PackageInfo $packageInfo
+        },
+        @{
+            "Package" = "Android SDK Build-tools"
+            "Version" = Get-AndroidBuildToolVersions -PackageInfo $packageInfo
+        },
+        @{
+            "Package" = "Google APIs"
+            "Version" = Get-AndroidGoogleAPIsVersions -PackageInfo $packageInfo
+        },
+        @{
+            "Package" = "Android Support Repository"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android Support Repository"
+        },
+        @{
+            "Package" = "Google Play services"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Google Play services"
+        },
+        @{
+            "Package" = "Google Repository"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Google Repository"
+        },
+        @{
+            "Package" = "SDK Patch Applier v4"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "SDK Patch Applier v4"
+        },
+        @{
+            "Package" = "CMake"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "cmake"
+        },
+        @{
+            "Package" = "NDK"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "ndk-bundle"
+        }
+    ) | Where-Object { $_.Version } | ForEach-Object {
+        [PSCustomObject] @{
+            "Package Name" = $_.Package
+            "Version" = $_.Version
+        } 
+    }
+}
+
+function Get-AndroidPackageVersions {
+    param (
+        [Parameter(Mandatory)]
+        [object] $PackageInfo,
+        [Parameter(Mandatory)]
+        [object] $MatchedString
+    )
+
+    $versions = $packageInfo | Where-Object { $_ -Match $MatchedString } | ForEach-Object {
+        $packageInfoParts = Split-TableRowByColumns $_
+        return $packageInfoParts[1]
+    }
+    return ($versions -Join "<br>")
+}
+
+function Get-AndroidPlatformVersions {
+    param (
+        [Parameter(Mandatory)]
+        [object] $PackageInfo
+    )
+
+    $versions = $packageInfo | Where-Object { $_ -Match "Android SDK Platform " } | ForEach-Object {
+        $packageInfoParts = Split-TableRowByColumns $_
+        $revision = $packageInfoParts[1]
+        $version = $packageInfoParts[0].split(";")[1]
+        return "$version, (rev $revision)"
```

Comment:
Let's remove `,` here. It looks pretty strange:
![image](https://user-images.githubusercontent.com/16715858/93304510-cca34b00-f805-11ea-8a25-afaba4a0efbf.png)